BM6toDB/BM6toDB.cpp

```diff
@@ -5,6 +5,10 @@
 #include <vector>
 #include <unordered_map>
 
+#ifdef LINUX_BUILD
```

Comment:
what?  Why is this only LINUX_BUILD ?
cstring is a stdandard c++ header

Reply:
I wrapped any change that I made in LINUX_BUILD so that it wouldn't affect windows. Some were more necessary than others. From my understanding, LINUX_BUILD was defined during cmake, so those portions were only run with cmake. For some reason, cstring wasn't necessary to include in the windows build, and I'm not sure why that is.